packages/smart-wallet/src/smartWallet.js

```diff
@@ -972,13 +972,14 @@ export const prepareSmartWallet = (baggage, shared) => {
 
             const invitation = invitationFromSpec(offerSpec.invitationSpec);
 
-            const [paymentKeywordRecord, invitationAmount] = await Promise.all([
+            const invitationAmount =
+              await E(invitationIssuer).getAmountOf(invitation);
```

Comment:
Looks like we're reconsidering the design from...

 - https://github.com/Agoric/agoric-sdk/issues/7098

Is there anybody we should check with? The record only shows you, me, and @turadg as party to that issue and its fix.